cmd/node/config/ratings.toml

```diff
@@ -0,0 +1,34 @@
+#Ratings configuration
+
+[General]
+    StartRating = 500000
+    MaxRating = 1000000
+    MinRating = 1
+    SelectionChances = [
+        { MaxThreshold = 0, ChancePercent = 5},
+        { MaxThreshold = 100000,ChancePercent = 0},
+        { MaxThreshold = 200000,ChancePercent = 16},
+        { MaxThreshold = 300000,ChancePercent = 17},
+        { MaxThreshold = 400000,ChancePercent = 18},
+        { MaxThreshold = 500000,ChancePercent = 19},
+        { MaxThreshold = 600000,ChancePercent = 20},
+        { MaxThreshold = 700000,ChancePercent = 21},
+        { MaxThreshold = 800000,ChancePercent = 22},
+        { MaxThreshold = 900000,ChancePercent = 23},
+        { MaxThreshold = 1000000,ChancePercent = 24},
+    ]
+
+[ShardChain.RatingSteps]
+    ProposerIncreaseRatingStep = 405
```

Comment:
Maybe add some comments here (containing succint formulas) as to be able to compute the values on different testnet configurations

Reply:
added to the begining of rating.toml